### PR TITLE
Home page slides

### PR DESCRIPTION
**Pull request by tomekwoojcik**

### CODE REVIEW

File: src/pages/home-page-slides/home-page-slides-context.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+interface SlidesContextModel {
+    carouselArr: CarouselModel[];
+    idxObj:number,
+    setIdxObj:Dispatch<React.SetStateAction<number>>
+}
+
+export const SlidesContext = createContext({} as SlidesContextModel);
+
+export const SlidesProvider = ({ children }: PropsContextModel) => {
```
Comment (liftoff-project) [inline src/pages/home-page-slides/home-page-slides-context.tsx]: Why do we need context for slides on the home page?

File: src/context/footer-context.tsx [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 interface FooterContextModel {
   categories: CategoriesModel[] | undefined;
   helpCenterArr: MenuBoxObjModel[];
-  setHandleEmail: React.Dispatch<React.SetStateAction<string>>;
+  setHandleEmail: Dispatch<SetStateAction<string>>;
   getEmail: () => void;
+  breakpointView:boolean;
 }
 
 export const FooterContext = createContext({} as FooterContextModel);
```
Comment (liftoff-project) [inline src/context/footer-context.tsx:16]: I know this code has already been there, but I've just noticed it now. Why do we need context for the footer?

File: src/model/props-context-model.tsx [hunk 3]
```diff
@@ -1,3 +1,3 @@
-export interface PropsModel{
+export interface PropsContextModel{
```
Comment (liftoff-project) [inline src/model/props-context-model.tsx]: I think it doesn't make sense to separate out into a separate file a type that is actually just one-liner. I would leave declarations of this type in the context file. In the "model" folder, I would keep domain models such as product, basket, etc.

**Comment by liftoff-project** [comment]
please resolve conflicts here.

File: src/pages/home-page-slides/home-page-slides-context.tsx (hunk 1 above)
Comment (tomekwoojcik) [reply src/pages/home-page-slides/home-page-slides-context.tsx]: this code is no actual. In github is new branch where i remove all small context in project.

File: src/context/footer-context.tsx (hunk 2 above)
Comment (tomekwoojcik) [reply src/context/footer-context.tsx:16]: this code is no actual. In github is new branch where i remove all small context in project.

File: src/pages/home-page-slides/home-page-slides-context.tsx (hunk 1 above)
Comment (tomekwoojcik) [reply src/pages/home-page-slides/home-page-slides-context.tsx]: this code is no actual. In github is new branch where i remove all small context in project.

File: src/model/props-context-model.tsx (hunk 3 above)
Comment (tomekwoojcik) [reply src/model/props-context-model.tsx]: at the moment where, i remove context, this model is not required

File: src/context/footer-context.tsx (hunk 2 above)
Comment (tomekwoojcik) [reply src/context/footer-context.tsx:16]: this code is no actual. In github is new branch where i remove all small context in project.

**Comment by tomekwoojcik** [comment]
all conflicts is resolved

**Review by liftoff-project — APPROVED** [review]

**Pull request merged**